db/migrate/20200312175018_create_blacklisted_referrers.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBlacklistedReferrers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :blacklisted_referrers do |t|
+      t.string :prefix, null: false
```

Comment:
I also wonder if we do the change I suggested above, it if makes more sense to do `host_with_path` or something to help denote what we want to store.

Reply:
Yep, good call.